### PR TITLE
Add styles

### PR DESCRIPTION
**Pull request by Roland-Ntwali**
## In this pull request, I did the following:

- Add styles to the page
- Fix issue of adding empty books

### CODE REVIEW

File: src/components/Render.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import Addbook from './Addbook';
+import Booklist from './Booklist';
+
+const Renderbooks = () => (
+  <>
+    <Booklist />
+    <Addbook />
+  </>
```
Comment (Olamarx) [inline src/components/Render.js:8]: - [ ] I will implore you to make sure each book card has a progress bar and all other content is present just as it is in the design. I also noticed the ul list-style is present in your design, kindly remove it also.

### Design Template
image

### Your Design
image

File: src/components/header.js [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
+import '../styles.css';
 
 const Header = () => (
   <div className="header">
     <h1>Bookstore CMS</h1>
-    <Link to="/">Books</Link>
-    <Link to="/categories">Categories</Link>
+    <Link className="BooksH" to="/">Books</Link>
+    <Link className="CategoriesH" to="/categories">Categories</Link>
   </div>
```
Comment (Olamarx) [inline src/components/header.js]: - [ ] You will notice there is a profile-like image at the right hand of the template design, kindly add it to yours too. Also, I would encourage you to style the books and categories links to fit in the template as they are close to each other in the template and both are close to the logo unlike what is in yours at the moment.

### Your design
image

### Design template
image

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @Roland-Ntwali, 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Linters passed ✔️
- Industry's standard styling approach/method ✔️
- Readme file professional ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Booklist.js [hunk 3]
```diff
@@ -1,21 +1,29 @@
...
-          id={book.item_id}
-        />
+        <li key={book.item_id}>
+          <CreateBook
+            title={book.title}
+            author={book.author}
+            id={book.item_id}
+
+          />
+        </li>
```
Comment (brainconnect93) [inline src/components/Booklist.js:24]: - [ ] As suggested by the previous reviewer, I would kindly advise you to remove the list-style-type dot showing in your design as this makes your design looks a little bit different from the original zeplin design.
You can set the list-style-type property to `none`

Your Design 👇 

image

**Review by brainconnect93 — CHANGES_REQUESTED** [review]
### Hi @Roland-Ntwali, 👋 

# STATUS: Changes Required :recycle:

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To Highlight :1st_place_medal: 

- Descriptive Pull Request title and summary. ✔️
- Styling of the bookstore CMS project ✔️
- Zeplin Design template used for the design ✔️
- There is a good use of Git flow ✔️

### Changes Required

- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @brainconnect93 ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sumairq — APPROVED** [review]
### Hi @Roland-Ntwali , 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Git flow
✔️Descriptive PR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**